README.md

```diff
@@ -130,6 +130,32 @@ GET my-index/doc/1
 
 GET my-index/doc/2
 ```
+During text processing, the plugin clears it. In particular, it cuts out e-mails and links to sites. In addition, the text may contain only numbers and it is impossible to determine the language. In this case, an exception is thrown. This may interfere with bulk queries or reindex queries. A similar situation can be handled like this:
+```
+{
+  "description": "Langdetect on body2.txt.",
+  "processors": [
+  
+    {
+      "langdetect": {
+        "if": "ctx.body2?.txt != null && ctx.body2.txt != \"\"" ,
```

Comment:
have you tested this on 7.3.1 as it includes https://github.com/spinscale/elasticsearch-ingest-langdetect/commit/de9665a5d593df961d96d29854d396bf32b69a13 to handle this case more graceful?

Reply:
I use 6.x ES and 6.8.2.1 plugin. "IF" forgot to clean.
In theory, so "IF" is not needed at all. All exceptions will go to "on_failure"

Reply:
update